hushline/settings.py

```diff
@@ -176,12 +176,20 @@ def index() -> str | Response:  # noqa: PLR0911, PLR0912
 
             # Handle Change Password Form Submission
```

Comment:
Is this section needed if the ``change_password`` route function exists? I tried removing this section to see if the tests would pass, and they do. I'm not sure if that's because this isn't needed, or just because it isn't tested.

Reply:
Agreed, it looks like only changes to display name, username, directory visibility, and bio are handled via the `/settings` route, while 2fa, password changes, smtp settings, pgp settings, account deletion, verified user and admin user settings are handled by dedicated routes.